jobs/future-effective-filings/file_future_effective.py

```diff
@@ -106,12 +85,29 @@ async def run(loop, application: Flask = None):  # pylint: disable=redefined-out
             for filing in filings:
                 filing_id = filing['filing']['header']['filingId']
                 effective_date = filing['filing']['header']['effectiveDate']
+                filing_type = 'filing'
+                try:
+                    filing_type = filing['filing']['business']['legalType']
+                except:
+                    filing_type = filing['filing']['registration']['nameRequest']['legalType']
+                cloud_event = SimpleCloudEvent(
+                  source=__name__[: __name__.find(".")],
+                  subject="filing",
+                  type="Filing",
+                  data={
+                      "filingId": filing_id,
+                      "filingType": filing_type,
+                      "filingEffectiveDate": effective_date,
+                    },
+                )
                 # NB: effective_date and now are both UTC
                 now = datetime.utcnow().replace(tzinfo=timezone.utc)
                 valid = effective_date and parse(effective_date) <= now
                 if valid:
-                    msg = {'filing': {'id': filing_id}}
-                    await queue_service.publish(subject, msg)
+                    # Publish to new GCP Filer Q
+                    #filer_topic = current_app.config.get("ENTITY_FILER_TOPIC", "filer")
+                    #queue.publish(topic=filer_topic, payload=queue.to_queue_message(cloud_event))
```

Comment:
Docker now builds and runs successfully with this code commented out:
![docker runs ok](https://github.com/bcgov/lear/assets/122301442/4191f4ac-60d0-49cf-8d52-d0fac4865f8f)

However, with this code commented in:
![docker run next error](https://github.com/bcgov/lear/assets/122301442/2f00445d-afc4-416c-8931-a1893d2857a0)

There's an authentication error.